topics/introduction.adoc

```diff
@@ -12,12 +12,7 @@ Customer Content Services (CCS) is a Red Hat team responsible for producing docu
 
 In principle, single-streamed documentation will be worked on completely in the upstream and brought down to the Customer Portal wholesale. In some projects, this process eliminates the overhead of maintaining two separate repositories for documentation content. This framework guides you through the process necessary for deciding if a community is a viable single-stream partner, setting up your repositories, choosing the right tools, and more.
 
-[[ccg-evaluating-the-framework]]
-=== Evaluating the Framework
-
-Placeholder text.
-
-[[ccg-checklist]]
-==== Checklist
```

Comment:
Wrt #63, do we want to retain Checklist here?

Reply:
I think it has to be *Something* Checklist - and I also think it should probably come at the end - as if to say 'did you do all these things?'  Does that make sense?

Reply:
I usually want a checklist at the <i>beginning</i>, to give me an overview of what I'm getting myself into.

Reply:
Oh, okay :) I'm not picky :)